.github/workflows/yamllint.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+name: YAML Validation
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
On al pushes? I'd rather get the error on push to branch before push to master.

Reply:
Done. I think for such cases the PR check, which also provides errors in comments, is more useful.